.github/workflows/publish-maven.yml

```diff
@@ -0,0 +1,52 @@
+name: publish-maven
```

Comment:
I confirmed that this workflow can upload artifacts to the sonatype staging repository.
(though I don't know if it actually `closeAndReleaseRepository`)

https://github.com/yshrsmz/android-client-sdk/actions/runs/3140792650/jobs/5102548182